src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -73,5 +73,23 @@ public static IEnumerable<Operation> GetOperationsByRequestMethod(string id, Ser
         {
             return serviceDefinition.Paths.Values.Select(pathObj => pathObj.Where(pair=> pair.Key.ToLower().Equals(id.ToLower()))).SelectMany(pathPair => pathPair.Select(opPair => opPair.Value));
         }
+
+        public static IEnumerable<string> GetResponseModelDefinitions(ServiceDefinition serviceDefinition)
```

Comment:
Please add comments stating that it only gets definitions for 200 responses not other status codes.